sw/device/sca/aes_serial.c

```diff
@@ -231,6 +269,15 @@ static void aes_serial_batch_encrypt(const uint8_t *data, size_t data_len) {
   SS_CHECK(data_len == sizeof(num_encryptions));
   num_encryptions = read_32(data);
 
+  block_ctr += num_encryptions;
+  // Rewrite the key to reset the internal block counter. Otherwise, the AES
+  // peripheral might trigger the reseeding of the internal masking PRNG which
+  // disturbs SCA measurements.
+  if (block_ctr > kBlockCtrMax) {
+    aes_serial_set_key(key_fixed, kAesKeyLength);
```

Comment:
`aes_serial_key_set` and `aes_serial_set_key` seem easy to get confused :) How about s/`aes_serial_set_key`/`aes_start`/? Since the functions are static, they don't need to have the module prefix name.

Reply:
Yes, you are right. I will do another commit to change this. There are actually some more functions that could be static but still have the `aes_serial` in their name which is confusing. I will change this.

Reply:
This is now done as well.